docs/src/use/configure/ignore.md

```diff
@@ -78,6 +78,33 @@ export default [
 ];
 ```
 
+Note that a pattern like `build/**` matches directory `build` and its contents, whereas `build/**/*` matches only its contents. Therefore, if you want to ignore the entire contents of a directory except for some parts, the latter pattern may be more practical. For example, this config ignores all contents of a `build` directory except for `build/test.js`:
+
+```js
+export default [
+    {
+        ignores: [
+            "build/**/*",               // ignore all contents in and under `build/` directory but not the `build/` directory itself
+            "!build/test.js"            // unignore `!build/test.js`
+        ]
+    }
+];
+```
+
+Patterns that end with `/` can match only directories. Patterns that don't end with `/` can match both files and directories. For example, this config ignores all files in and under `build` directory except for files named `test.js`:
```

Comment:
This example is really difficult to follow. I don't think this paragraph properly introduces what is happening in the config below. Can you make this more explicit?

Reply:
I've now pushed a new version with more details about the example.